net-utils/src/bin/ip_address_server.rs

```diff
@@ -21,7 +22,10 @@ fn main() {
         .unwrap_or_else(|_| panic!("Unable to parse {port}"));
     let bind_addr = SocketAddr::from((Ipv4Addr::UNSPECIFIED, port));
     let tcp_listener = TcpListener::bind(bind_addr).expect("unable to start tcp listener");
-    let _runtime = solana_net_utils::ip_echo_server(tcp_listener, /*shred_version=*/ None);
+    let num_threads =
+        std::num::NonZeroUsize::new(DEFAULT_IP_ECHO_SERVER_THREADS).expect("non-zero num threads");
```

Comment:
assume we're not using `.unwrap_or(NonZeroUsize::new(2))` here since it's just a standalone `main` function, unlike `discover()` in `gossip_service.rs`

Reply:
I honestly went a little back and forth on this, and as a result, was inconsistent 😅 . Given that I added this:
```rust
// There must be at least one worker so the value must be non-zero
static_assertions::const_assert!(DEFAULT_IP_ECHO_SERVER_THREADS > 0);
```
I'm actually leaning towards doing the `.expect()` directly in gossip_service as well.